modules/juce_graphics/geometry/juce_Point.h

```diff
@@ -41,32 +45,32 @@ class Point
 {
 public:
     /** Creates a point at the origin */
-    Point() noexcept : x(), y() {}
+    constexpr Point() noexcept : x(), y() {}

```

Comment:
The JUCE team already added a series of changes like this on `develop`: https://github.com/WeAreROLI/JUCE/commit/4a2a5294a7bca077bedf9f68e4774075ac8843f8